h2o-algos/src/main/java/hex/tree/SharedTreeModel.java

```diff
@@ -488,6 +492,8 @@ protected Frame execute(Frame adaptFrm, String[] names, Key<Frame> destKey) {
     private transient int[/*treeId*/][/*classId*/] _maxNodeIds;
     // OUT
     private double[/*treeId*/][/*classId*/][/*leafNodeId*/] _leafNodeWeights;
+    private int[] _warn_trees;
```

Comment:
I would use camelCase to be in consistent with previous variables 😇

Reply:
yes, here it is inconsistent